cli/inspector.rs

```diff
@@ -415,7 +415,11 @@ impl DenoInspector {
     });
 
     // Tell the inspector about the global context.
-    let context = core_state.global_context.get(scope).unwrap();
+    let context = core_state
+      .global_context
+      .as_ref()
+      .map(|context| v8::Local::new(scope, context))
```

Comment:
I see that in some cases it's a bit unpractical that Global::get() now deferences the handle rather than creating a new Local (which is what it used to do). I could add a method like `Global::to_local()` to get some of that convenience back.